docs/src/gettingstarted/introduction/own-code/cli-install.md

```diff
@@ -6,52 +6,21 @@ aliases:
   - "/gettingstarted/own-code/cli-install.html"
 ---
 
-In the previous steps, you checked that the requirements on your computer were met and configured an SSH key on your Platform.sh account. Now all we have to do is install the CLI and you can access your projects from the command line.
+In the previous steps, you checked that the requirements on your computer were met. Now all we have to do is install the CLI and you can access your projects from the command line.
 
 {{< asciinema src="videos/asciinema/verify-cli-extended.cast" >}}
 
-1. **Install the CLI**
+##  1. Install the CLI
 
-    In your terminal, run the following command depending on your operating system:
+{{% cli-installation %}}
 
-{{< codetabs >}}
----
-title=Linux/macOS
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -fsS https://platform.sh/cli/installer | php
-
-<--->
-
----
-title=Windows
-file=none
-highlight=bash
-markdownify=false
----
-
-curl.exe -f https://platform.sh/cli/installer -o cli-installer.php
-php cli-installer.php
-
-{{< /codetabs >}}
-
-
-1. **Authenticate and verify**
-
-   Once the installation has completed, run the CLI in your terminal with the command:
-
-   ```bash
-   platform
-   ```
+##  2. Test the CLI
 
-   Log in using a browser. Then take a moment to view some of the available commands with the command:
+Get a list of all available commands with:
 
-   ```bash
-   platform list
-   ```
+```bash
+platform list
+```
```

Comment:
:lady_beetle: Given that the heading is "test", it should maybe include logging in. This command doesn't include that.